### PR TITLE
Front/Set up SCSS

### LINKED ISSUE #4: Set up SCSS
SCSS is a superset of CSS.

https://github.com/webpack-contrib/sass-loader

### PR DESCRIPTION
**Pull request by bwdmonkey**
:tickets: **Ticket(s)**: Closes #4 

---

## :construction_worker: Changes

Set up SCSS for frontend

## :flashlight: Testing Instructions

```
cd frontend/
npm install
npm start
```
Open up `localhost:3000` and see if your eyes hurt or not. If it doesn't, something is wrong with my code or with you.

### CODE REVIEW

File: frontend/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$least-popular-color-in-the-world: #FF0;
```
Comment (bobheadxi) [inline frontend/src/index.scss]: 🤢

**Review by bobheadxi — DISMISSED** [review]
awesome work, thanks!

**Comment by bobheadxi** [comment]
works for me, but oh god

image

**Review by bobheadxi — DISMISSED** [review]
thanks for comma

**edit** jk u dun goof again

**Review by bobheadxi — APPROVED** [review]
travis is happy

**Review by omarkhs — COMMENTED** [review]
works for me too!

screen shot 2018-10-04 at 1 04 39 am

**Comment by bobheadxi** [comment]
@omarkhs actually that's not quite right - it should be an ugly shade of yellow like in my screenshot 🤔 

@leesw98 any ideas?

**Comment by omarkhs** [comment]
@bobheadxi hmm I am not sure why but fyi here is a screenshot of what shows up on my terminal after I run commands that tom mentioned above, so it seems to compiles successfully.

screen shot 2018-10-04 at 6 34 04 pm

**Comment by leesw98** [comment]
@omarkhs Can you try

```
npm build
npm start
```

**Comment by omarkhs** [comment]
I used npm build but I get this 

screen shot 2018-10-04 at 6 44 08 pm

also I think I may know why the I am not seeing that yellow, for some reason I didn't fetch scss files, I only have css. 
 
Here is are files in my frontend/src directory:

screen shot 2018-10-04 at 6 47 27 pm

**Comment by bobheadxi** [comment]
@omarkhs you'll have to checkout the branch that of this PR - in this case, `front/set-up-scss`, like so:

```
git checkout front/set-up-scss
```

**Comment by omarkhs** [comment]
oh shooot! I missed that...sorry :/

**Comment by bobheadxi** [comment]
haha no worries! glad to know it worked

**Pull request merged**